### ISSUE TITLE
Not enough disk space?

### ISSUE THREAD

**Issue by NovenTheHero** [dangiu/PicoMemcard#26]
I am trying to put the blank memory card file on to the pico and it says there is not enough free disk space. Super weird!
![image](https://user-images.githubusercontent.com/3946547/188932539-de9b27ec-3335-457c-9f77-0492f446c1ce.png)

**Comment by NovenTheHero** [comment]
![image](https://user-images.githubusercontent.com/3946547/188933423-e5960ba4-ceef-4c28-8847-c6b13693e1bc.png)

**Comment by Androxilogin** [comment]
You're overwriting the MEMCARD.MCD file, correct? Just delete it and throw on the new file.

**Comment by NovenTheHero** [comment]
No, just installing the firmware, then dragging the memory card file on to the folder.

**Comment by dangiu** [comment]
Sometimes windows automatically puts in garbage data to keep track of some stuff... I specifically enlarged the disk above 128KB to make sure this is not a problem but apparently it still seems to be!
Could you test something for me? Can you download the nuke_flash.utf2 from [this page](https://www.raspberrypi.com/documentation/microcontrollers/raspberry-pi-pico.html#resetting-flash-memory) and flash it onto the raspberry to ensure that all the data gets wiped. Then reinstall PicoMemcard and check if the problem is still there

**Comment by NovenTheHero** [comment]
I did that and I am getting the same results. This is super weird, and mildly frustrating.

**Comment by dangiu** [comment]
You are on Windows 10 correct? Could you try to repeat the same procedure (nuking the Pico flash memory) and flash PicoMemcard from a different os? (maybe a live linux distro such as ubuntu). I'm sorry but I can't to much to debug the issue since it seems to be computer specific.
If anybody else is having this problem please report here your machine configuration.

**Comment by NovenTheHero** [comment]
Super weird, I did the nuke, then reflashed it on my linux machine and I think it worked. I will test it out on real hardware later when I get home from work, but tentatively, it is good. At least no issues copying the memcard.

**Comment by arkwise** [comment]
hmm weird.. ive tried Mac + linux and windows and the picoMC device shows up but there is no partition or anything,

I flashed the uf2 file fine and the picoMC drive shows up afterwards but there is no partition at all just the device. 

when you try to open it or drop file it just says please insert removable media (win) and Mac and linux just say there is no drive/partition available.

I then re-bootsel and dropped my mcr's into this and the uf2 last and all it flashes but no memory card images are available.

**Comment by dangiu** [comment]
Which version did you install? You need the plus version for using the micro sd (and of course a micro sd extension and the micros sd itself)

**Comment by arkwise** [comment]
> Which version did you install? You need the plus version for using the micro sd (and of course a micro sd extension and the micros sd itself)

the standard version I am not using a SD CARD for storage.

**Comment by dangiu** [comment]
> Sometimes windows automatically puts in garbage data to keep track of some stuff... I specifically enlarged the disk above 128KB to make sure this is not a problem but apparently it still seems to be! Could you test something for me? Can you download the nuke_flash.utf2 from [this page](https://www.raspberrypi.com/documentation/microcontrollers/raspberry-pi-pico.html#resetting-flash-memory) and flash it onto the raspberry to ensure that all the data gets wiped. Then reinstall PicoMemcard and check if the problem is still there

Have you already tried to to this?

**Comment by arkwise** [comment]
> > Sometimes windows automatically puts in garbage data to keep track of some stuff... I specifically enlarged the disk above 128KB to make sure this is not a problem but apparently it still seems to be! Could you test something for me? Can you download the nuke_flash.utf2 from [this page](https://www.raspberrypi.com/documentation/microcontrollers/raspberry-pi-pico.html#resetting-flash-memory) and flash it onto the raspberry to ensure that all the data gets wiped. Then reinstall PicoMemcard and check if the problem is still there
> 
> Have you already tried to to this?

sorry for the late reply, i get an error 1407 on my mac, now i have about 10 of these unit and i get the same results and theyre official units and ive purchased them at different times over the past few years and function fine for other projects so i know theyre mint and ive tried the nuke flash and both version of pico meem to no avail.  the only difference is the nuke remounts the storage after i loaded nuke onto it but as soon as i try to load pico mem its back to square one.

im so keen on this for another project im working on so any help is appreciated mate :)

**Comment by dangiu** [comment]
Can you try on a different system? Possibily some linux distribution, not on an apple machine
